File: src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Conservation/WaterConservationApplicationRecommendationRequest.cs
using WesternStatesWater.WestDaat.Common.DataContracts;

namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Conservation;

public class WaterConservationApplicationRecommendationRequest : ApplicationStoreRequestBase
{
    public Guid WaterConservationApplicationId { get; set; }
    
    public RecommendationDecision RecommendationDecision { get; set; }

    public string RecommendationNotes { get; set; }
}

File: src/DashboardUI/src/data-contracts/WaterConservationApplicationRecommendationRequest.ts
import { ApplicationStoreRequestBase } from './ApplicationStoreRequestBase';
import { RecommendationDecision } from './RecommendationDecision';

export interface WaterConservationApplicationRecommendationRequest extends ApplicationStoreRequestBase {
  $type: 'WaterConservationApplicationRecommendationRequest';
  waterConservationApplicationId: string;
  recommendationDecision: RecommendationDecision;
  recommendationNotes?: string;
}
